server/text_generation_server/layers/gptq/__init__.py

```diff
@@ -109,6 +116,10 @@ def get_weights(self, weights: Weights, prefix: str):
             log_once(logger.warning, "Disabling exllama because desc_act=True")
             use_exllama = False
 
+        if self.is_layer_skipped_quantization(prefix, self.modules_to_not_convert):
+            w = weights.get_tensor(f"{prefix}.weight")
```

Comment:
I think it's nicer to call into `DefaultWeightsLoader` in all these skip cases, then we only have to modify code in one place when there are changes in the loading code.

Reply:
Done